### ISSUE TITLE
Docker Desktop - UI for managing volumes not working

### ISSUE THREAD

**Issue by dobri-002** [docker/for-win#12789]
Open Docker Desktop and navigate to "Volumes" tab. A loading icon appears and it keeps rotating forever - nothing happens and no error message is displayed. You can see it in the attached video.

In Windows Event Viewer I see a few messages like these:
**_Failed to determine size of volume [error=open C:\ProgramData\Docker\volumes\{GUID}\_data\mnesia: Access is denied. volume={GUID}]_**

They are most probably written when Docker Desktop starts and I cannot be sure that they are related with this issue. 
The problem is reproducible when I run Docker Desktop both as Administrator and as an ordinary user.

![docker-desktop-volumes-tab](https://user-images.githubusercontent.com/16239003/174673263-13398387-f5bc-481a-94e1-31d60b1be425.gif)

Windows 21H2 Build 22000.739
Docker Desktop 4.9.1 (81317)

```
docker version

Client:
 Cloud integration: v1.0.25
 Version:           20.10.16
 API version:       1.41
 Go version:        go1.17.10
 Git commit:        aa7e414
 Built:             Thu May 12 09:17:07 2022
 OS/Arch:           windows/amd64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.9.1 (81317)
 Engine:
  Version:          20.10.16
  API version:      1.41 (minimum version 1.24)
  Go version:       go1.17.10
  Git commit:       f756502
  Built:            Thu May 12 08:44:17 2022
  OS/Arch:          windows/amd64
  Experimental:     true
```

```
docker context show

default
```

```
docker info

Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc., v0.8.2)
  compose: Docker Compose (Docker Inc., v2.6.0)
  sbom: View the packaged-based Software Bill Of Materials (SBOM) for an image (Anchore Inc., 0.6.0)
  scan: Docker Scan (Docker Inc., v0.17.0)

Server:
 Containers: 16
  Running: 0
  Paused: 0
  Stopped: 16
 Images: 65
 Server Version: 20.10.16
 Storage Driver: windowsfilter (windows) lcow (linux)
  Windows:
  LCOW:
 Logging Driver: json-file
 Plugins:
  Volume: local
  Network: ics internal l2bridge l2tunnel nat null overlay private transparent
  Log: awslogs etwlogs fluentd gcplogs gelf json-file local logentries splunk syslog
 Swarm: inactive
 Default Isolation: hyperv
 Kernel Version: 10.0 22000 (22000.1.amd64fre.co_release.210604-1628)
 Operating System: Windows 10 Pro Version 2009 (OS Build 22000.739)
 OSType: windows
 Architecture: x86_64
 CPUs: 16
 Total Memory: 31.91GiB
 Name: ***REDACTED***
 ID: ***REDACTED***
 Docker Root Dir: C:\ProgramData\Docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: true
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine
```

**Comment by trungutt** [comment]
Hi @robbie-coder we're trying to look into this issue. I followed two .zip templates given in https://github.com/docker/compose/issues/9577, but couldn't reproduce the issue on my side. To make sure that we understand correctly the problem, could you please follow these instructions:

- Reproduce the infinite loading and tell us how to reproduce it with a minimum steps (how to do, which compose version...)
- Upload a diagnostic and share its Id (https://docs.docker.com/desktop/windows/troubleshoot/#diagnose-and-feedback)

Thanks for reporting

**Comment by robbie-coder** [comment]
@trungutt I can reproduce it with these steps (now used v4.10.1):

- In settings make sure that _Use Docker Compose v2_ is unchecked
- In windows container mode using the [docker-compose.zip](https://github.com/docker/for-win/files/9071786/docker-compose.zip) (same as before) do a **docker-compose up** from a terminal, note the - between docker and compose
- The volumes tab in docker dashboard now works fine.
- Quit docker desktop (right click icon in systray select Quit Docker Desktop
- Start docker desktop again
- Now the volumes tab in Dashboard shows the spinning icon

Interestingly if you don't quit docker desktop but select restart from the same context menu the problem does not occur.

When the volumes tab is unresponsive I can only fix it by switching to Linux containers mode and then switch back to Windows containers mode, then the volumes tab works again for a while.

Diagnostics id: 05424DC1-A82A-4E44-9801-B269BD475592/20220708114313

Let me know if you need more information.

**Comment by trungutt** [comment]
Right I followed these steps and reproduced the same issue. Thanks a lot for your instructions, we're trying to figure out how to handle this annoying issue

**Comment by trungutt** [comment]
Hi @robbie-coder could you please try out this build

https://desktop-stage.docker.com/win/main/amd64/83017/Docker%20Desktop%20Installer.exe

to see if the problem still persists?

**Comment by robbie-coder** [comment]
@trungutt with that version I can no longer reproduce the problem.

Thanks for the fix.

**Comment by gavin-timbrell** [comment]
We've just released [Docker Desktop 4.11.0](https://docs.docker.com/desktop/release-notes/#docker-desktop-4110) which includes a fix for this.
